arches/app/datatypes/url.py

```diff
@@ -255,3 +264,7 @@ def default_es_mapping(self):
                 },
             }
         }
+
+    def pre_tile_save(self, tile, nodeid):
+        if tile and "url_label" not in tile.data[nodeid]:
```

Comment:
Will also need to guard against `tile.data[nodeid]` being None:

```
{
    "message": "argument of type 'NoneType' is not iterable",
    "status": "false",
    "success": false,
    "title": "Unable to save."
}
```

Reply:
Ah, this occurs when an empty url datatype tile is saved, something I forgot to test.  Great spot.  

I have added an `if tile.data[nodeid]:` in [2aedaa7](https://github.com/archesproject/arches/pull/10642/commits/2aedaa718b484c95f92de69003cb69747536ab9f), would this suffice?